Docs/wfldocs/WFL-io.md

```
@@ -314,6 +326,384 @@ In this snippet, `perform fetch from url` starts the HTTP GET requests. We don
+shell_execution_mode = "forbidden"    # Most secure (default)
+warn_on_shell_execution = true
+allowed_shell_commands = []
+
+# Available modes:
+# "forbidden"       - No shell execution allowed (recommended)
+# "allowlist_only"  - Only commands in allowed_shell_commands can use shell
+# "sanitized"       - Shell allowed with validation and warnings
+# "unrestricted"    - Legacy mode (NOT recommended for production)
+```
+
+**Example: Enabling Shell with Warnings**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)